app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/_module.less

```diff
@@ -813,6 +813,7 @@
             &.delete {
```

Comment:
still one line around 457 have style on delete button. You should clean that to avoid confusing in future
You can indicate that line with property right: 0;